tests/benchdnn/softmax/softmax.cpp

```diff
@@ -225,9 +225,10 @@ void skip_invalid_prb(const prb_t *prb, res_t *res) {
 void setup_cmp(compare::compare_t &cmp, const prb_t *prb, data_kind_t kind,
         const args_t &ref_args) {
     const auto trh_dt = (prb->dir & FLAG_FWD) ? prb->ddt : prb->sdt;
-    const bool is_flt_or_dbl = trh_dt == dnnl_f32 || trh_dt == dnnl_f64;
+    const bool is_fp
+            = trh_dt == dnnl_f32 || trh_dt == dnnl_f64 || trh_dt == dnnl_f16;
```

Comment:
Please elaborate on "Threshold for comparison of half precision floating point results was set wrong."

The change was done on purpose to catch issues risen due to not storing intermediate results into f32 data type buffers. If proper intermediate storing is not happening for AMD, that must be pointed out and carefully explained.

Reply:
Fixed.

Reply:
Sorry when reverting it, I missed this bit. Now it should actually be fixed.